src/App.tsx

```diff
@@ -1,24 +1,144 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
Consider removing the unused import of 'React' since you are already importing 'useEffect' and 'useState' from 'react'.